app/assets/javascripts/incidents/batch_export.coffee

```diff
@@ -0,0 +1,29 @@
+handleExporting = ->
+  inSelectMode = $('table.incidents th.batch-export').is ':visible'
+  if inSelectMode
+    ids = $('.batch-export input:checked').map ->
+      $(this).data 'id'
+    .toArray()
+    alreadyExported= $('.batch-export input:checked').filter ->
+      $(this).data 'exported'
+    if alreadyExported.length > 0
+      if alreadyExported.length == 1
+        warning = '1 selected incident has already been exported.'
+      else warning = "#{alreadyExported.length} selected incidents have already been exported."
+      warning += "\nPlease doublecheck that you would like to re-export this incident."
```

Comment:
"this incident" or "these incidents"?

If you decide to be consistent with your pluralization, maybe you could abstract this out. I've had this problem before and haven't stumbled upon a compact solution.

Reply:
Good catch, thank you.